Azure/azure-cli
# [Network] Fix #23525: `az network bastion create/update`: Add missing arguments and update command

**Related command**


```sh
az network bastion create
az network bastion update
```

**Description**


Azure Bastion Hosts have a few configurations available from the Azure Portal but are missing in Azure CLI. This PR adds those configuration arguments and an update command as well.

resolves #23525, resolves #22595

**Testing Guide**


```sh
az network bastion create --name {} --resource-group {} --enable-native-client-support --disable-copy-paste --enable-ip-connect
az network bastion update --name {} --resource-group {} --enable-native-client-support --disable-copy-paste --enable-ip-connect
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.
